src/main/java/be/seeseemelk/mockbukkit/inventory/InventoryMock.java

```diff
@@ -122,34 +111,38 @@ public void assertContainsAtLeast(@NotNull ItemStack item, int amount)
 	 * @param item The item to check for.
 	 * @return The number of times the item is present in this inventory.
 	 */
-	public int getNumberOfItems(@NotNull ItemStack item)
-	{
+	public int getNumberOfItems(@NotNull ItemStack item) {
 		int amount = 0;
-		for (ItemStack itemstack : items)
-		{
-			if (itemstack != null && item.isSimilar(itemstack))
-			{
+		for (ItemStack itemstack : items) {
+			if (itemstack != null && item.isSimilar(itemstack)) {
 				amount += itemstack.getAmount();
 			}
 		}
 		return amount;
 	}
 
+	private Map<Integer, ItemStack> contentsToMap() {
+		Map<Integer, ItemStack> contents = new HashMap<>();
+
+		for (int i = 0; i < getSize(); i++) {
+			contents.put(i, items[i]);
+		}
+
+		return contents;
+	}
+
 	@Override
-	public int getSize()
-	{
+	public int getSize() {
```

Comment:
You broke the formatting here.

```suggestion
	public int getSize()
	{
```